src/clarity/forms/_forms.clarity.scss

```diff
@@ -258,34 +270,39 @@ $clr-custom-checkbox-radio-top: ($clr_baselineRem_1 - $clr-checkbox-radio-height
                 padding-left: 0;
                 margin-bottom: $clr_baselineRem_1;
 
-                & > label:first-of-type {
-                    position: relative;
+                & > label:first-child,
+                & > label:not(:first-child),
```

Comment:
Do both first-child and non-first child labels have the same style, ` flex: 1 1 100%;`? If then, I think we could just use `& > label`? I might be wrong here.